app/controllers/marks_controller.rb

```diff
@@ -0,0 +1,104 @@
+class MarksController < ApplicationController
+  before_action :results, only: [:vimeo_subscribe]
+  before_action :require_spy, only: [:index, :results, :show]
+
+  def twitter
+    Seemore::Application.config.twitter
+  end
+
+  def index
+    @marks = current_spy.marks
+  end
+
+  def results
+    if params[:username].present?
+      search_params = { username: params[:username], id: params[:id] }
+      @search_term = search_params[:username]
+      @vimeo_marks = Mark.vimeo_lookup(@search_term)
+      @twitter_marks = twitter_lookup(@search_term)
+    end
+  end
+
+  def twitter_lookup(search_term)
```

Comment:
See below comment on this kind of method being private or preferably in a helper class.